components/setup/TriggerContent.tsx

```diff
@@ -4,11 +4,18 @@ import Button from '../Button';
 interface TriggerContentProps {
     selectedProvider: string;
     bitbucket_auth_url: string;
+    selectedInstallationType: string;
+    selectedHosting: string;
 }
 
-const TriggerContent: React.FC<TriggerContentProps> = ({ selectedProvider, bitbucket_auth_url }) => {
+const TriggerContent: React.FC<TriggerContentProps> = ({ selectedProvider, bitbucket_auth_url, selectedHosting, selectedInstallationType }) => {
     const triggerContent = () => {
         if (selectedProvider === 'github') {
+            if (selectedHosting == 'selfhosting' && selectedInstallationType == 'individual'){
```

Comment:
Again, if you are adding the conditions, just add a "Coming soon" for individual+bitbucket.